src/libsystemd-network/sd-lldp.c

```diff
@@ -6,15 +6,25 @@
 #include "sd-lldp.h"
 
 #include "alloc-util.h"
+#include "ether-addr-util.h"
 #include "fd-util.h"
 #include "lldp-internal.h"
 #include "lldp-neighbor.h"
 #include "lldp-network.h"
 #include "socket-util.h"
-#include "ether-addr-util.h"
+#include "string-table.h"
 
 #define LLDP_DEFAULT_NEIGHBORS_MAX 128U
 
+static const char * const lldp_event_table[_SD_LLDP_EVENT_MAX] = {
+        [SD_LLDP_EVENT_ADDED]   = "add",
+        [SD_LLDP_EVENT_REMOVED] = "remove",
+        [SD_LLDP_EVENT_UPDATED]   = "update",
+        [SD_LLDP_EVENT_REFRESHED] = "refresh",
```

Comment:
i think the right side should close match the left side here, i.e. "added" instead of "add", and so on.